extism/go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/extism/cli/extism
 
 go 1.20
 
-replace github.com/extism/cli => ../
+// replace github.com/extism/cli => ../
```

Comment:
@nilslice - I had to comment out the `replace` directives in the go.mod files you added because they oddly don't work with `go install`: https://github.com/golang/go/issues/44840